plugins/ipam/host-local/host_local_test.go

```diff
@@ -517,6 +518,15 @@ var _ = Describe("host-local Operations", func() {
 	})
 })
 
+func getTmpDir() (string, error) {
+	tmpDir, err := ioutil.TempDir("", "host_local_artifacts")
+	if err == nil {
+		tmpDir = filepath.ToSlash(tmpDir)
```

Comment:
so without this the un-escaped backslashes in the windows path cause JSON parsing to barf